pages/token-group/permutations.page.tsx

```diff
@@ -179,6 +179,68 @@ const permutations = createPermutations<TokenGroupProps>([
       ],
     ],
   },
+  {
+    i18nStrings: [i18nStrings],
+    limit: [undefined],
```

Comment:
Similar to an earlier comment, do we need a permutation where there is a limit that has been exceeded and we are showing the "Show more" state?

Reply:
"Show more" behavior remains the same in this case, same as in the earlier comment